File: TicTacToe_Java_Python/sprint1_1/test/test_board_gui.py
import unittest

from sprint1_1.product.gui import GUI


class TestBoardGUI(unittest.TestCase):

    def setUp(self):
        pass

    def tearDown(self):
        pass

    def testEmptyBoard(self):
        GUI().drawEmptyBoard()


if __name__ == "__main__":
    unittest.main()

File: TicTacToe/src/sprint1_1/test/TestBoardGUI.java
package sprint1_1.test;

import org.junit.After;
import org.junit.Before;
import org.junit.Test;

import sprint1_1.product.Board;
import sprint1_1.product.GUI;

public class TestBoardGUI {
	private Board board;

	@Before
	public void setUp() throws Exception {
		board = new Board();
	}

	@After
	public void tearDown() throws Exception {
	}

	@Test
	public void testEmptyBoard() {
		new GUI(board);
		try {
			Thread.sleep(2000);
		} catch (InterruptedException e) {
			e.printStackTrace();
		}
	}

}
